contracts/interchain-token-service/src/contract.rs

```diff
@@ -50,6 +50,19 @@ impl InterchainTokenService {
             &interchain_token_wasm_hash,
         );
     }
+
+    fn set_token_id_config(env: &Env, token_id: BytesN<32>, token_data: TokenIdConfig) {
```

Comment:
avoid defining private methods inside `#[contractimpl]` impl blocks that are only meant for entrypoints